opensplice_cmake_module/env_hook/opensplice.bat.in

```diff
@@ -1,4 +1,27 @@
-set "OSPL_HOME=@OpenSplice_HOME@"
+set "_DEFAULT_OSPL_HOME=@OpenSplice_HOME@/"
+set "_DEFAULT_OSPL_HOME=%_DEFAULT_OSPL_HOME:/=\%"  :: Replace forward slashes with backslashes
```

Comment:
note: this is necessary because otherwise the string comparison fails below. e.g. for connext at the moment it's always printing in our CI jobs:
`"NDDSHOME changed from \"C:\Program Files\rti_connext_dds-5.3.0\" to \"C:/Program Files/rti_connext_dds-5.3.0\""`

Reply:
I would suggest to perform this conversion in CMake before expanding this template.